scoring/static/scoring/js/main.js

```diff
@@ -435,3 +435,40 @@ if ( typeof window.fetch !== 'undefined' ) {
         }
     });
 }
+
+// Graphic click tracking
+var trackEvent = function(eventName, params, callback){
+    params = params || {};
+    callback = callback || function(){};
+    params['event_callback'] = callback;
+    setTimeout(callback, 2000);
+    gtag('event', eventName, params);
+};
+
+// Function to extract section from the URL
+var getSectionFromURL = function() {
+    var urlParts = window.location.pathname.split('/');
```

Comment:
I’m slightly worried this is a bit fragile if we happened to change the URL pattern, and some missing data in GA would go easily unnoticed. Since we’re pulling lots of other data out of custom attributes on the `<a>` elements, maybe we should do the same here?